extensions/amp-json-schema/validator-amp-json-schema.protoascii

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-json-schema
+  html_format: AMP
+  html_format: EXPERIMENTAL
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-json-schema"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+
+tags: {  # amp-json-schema (json)
+  html_format: AMP
+  html_format: EXPERIMENTAL
+  tag_name: "SCRIPT"
+  spec_name: "amp-json-schema extension .json script"
+  unique: true
+  mandatory_parent: "HEAD"
+  requires_extension: "amp-json-schema"
+  attrs: {
+    name: "id"
+    mandatory: true
+    value: "amp-json-schema-premutate"
```

Comment:
I don't think we should validate this attribute value. This means that the `id` could only ever be "amp-json-schema-premutate".